### ISSUE TITLE
Felmeddelanden pga för många anrop?

### ISSUE THREAD

**Issue by fredrikbaberg** [fuffenz/ha-sensor-sl#2]
Hej,
Sker det någon utveckling på den här sensorn?

Min Home Assistant-installation har passerat 10.000 API-anrop den här månaden, varpå jag får en hel del felmeddelanden i loggen (antagligen ett per minut?). Jag tänkte försöka rätta till det, och undrar om du har några planerade ändringar på gång - så jag inte gör dubbelt arbete.

**Comment by fuffenz** [comment]
Ja, det är ett problem med API-begränsningen och att Trafiklab inte tillåter privatpersoner att uppgradera till nästa nivå.  Jag har tyvärr inte haft möjlighet att göra någon workaround än, så om du har några förslag är det bara att köra. :)

**Comment by fredrikbaberg** [comment]
Som du såg i tidigare PR så började jag med en mindre ändring (hade inte räknat med att den skulle bli accepterad utan vidare, men det var visserligen en liten ändring).

Jag har även påbörjat mer omfattande ändringar (https://github.com/fredrikbaberg/ha-sensor-sl), men det är en del kvar på det. Tänkte lägga några timmar till idag på att försöka få in fler funktioner, vet inte när jag kommer ha tid nästa gång.
Eftersom det blir en hel del ändringar mot nuvarande kod vet jag inte om det är vettigt att göra en PR (WIP, så den inte råkar godkännas) nu, eller om det ska hållas separat tills vidare? Du får gärna kolla på de issues jag skapat där (försöker hålla koll på vad för funktioner jag har lagt in och tänker mig skulle vara bra). Feature request #1 här försökte jag bland annat lägga in. Måste bara få fler anrop till API, just nu kan jag bara testa felhanteringen (som nästan fungerar som jag vill).
